package.json

```diff
@@ -59,7 +59,7 @@
     "moment": "~> 2.19.3",
     "noty": "3.1.4",
     "phantomjs-prebuilt": "^2.1.16",
-    "publiclab-editor": "publiclab/PublicLab.Editor#~1.3.0",
+    "publiclab-editor": "github:publiclab/PublicLab.Editor",
```

Comment:
I'd like to keep pegging this to a version - now `~1.4.0` maybe? Or even actually `^1.4.0` so we get minor updates too (`0.x.0`)?

Reply:
Done 👍